pkg/machines/components/vm/bootOrderModal.jsx

```diff
@@ -0,0 +1,319 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import cockpit from 'cockpit';
+import {
+    Button,
+    ButtonGroup,
+    Icon,
+    ListView,
+    ListViewItem,
+    Modal
+} from 'patternfly-react';
+
+import { ModalError } from './../notification/inlineNotification.jsx';
+import {
+    findHostNodeDevice,
+    getSortedBootOrderDevices,
+    rephraseUI,
+    vmId
+} from '../../helpers.js';
+import {
+    changeBootOrder,
+    getVm
+} from '../../actions/provider-actions.js';
+
+import './bootOrderModal.css';
+
+const _ = cockpit.gettext;
+const UP = -1;
+const DOWN = 1;
+
+function getBootOrderDevices(vm) {
+    let devices = getSortedBootOrderDevices(vm.inactiveXML);
+
+    devices.forEach(dev => {
+        dev.checked = typeof dev.bootOrder !== 'undefined';
+        dev.initialOrder = parseInt(dev.bootOrder);
+    });
+
+    return devices;
+}
+
+const DeviceInfo = ({ descr, value }) => {
+    return (
+        <div className='ct-form-layout'>
+            <label className='control-label' htmlFor={value}>
+                {descr}
+            </label>
+            <span id={value}>
+                {value}
+            </span>
+        </div>
+    );
+};
+
+const DeviceRow = ({ idPrefix, device, index, onToggle, upDisabled, downDisabled, moveUp, moveDown, nodeDevices }) => {
+    let heading;
+    let additionalInfo = [];
+
+    const addOptional = (additionalInfo, value, descr) => {
+        if (value) {
+            additionalInfo.push(<DeviceInfo
```

Comment:
Please put the <DeviceInfo in next row and there is not need to split into seperate lines the args any more.